1001-2000/LIT1898Martyr.xml

```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1898Martyr">
 
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Martyrium of Mamas, Tewodoṭos and his wife Tewofina</title>
+            <title xml:id="t1">Martyrium of Mamas, Tewodoṭos and his mother Tewofina</title>
```

Comment:
Tewodotos/Theodotus was Mamas' father, so Tewofina (Rufina in other traditions) was the latter's wife, so it was correct

If we are at correcting the record - do we translate the names or do we transcribe the names? (I guess either, in English, Mamas/Mammes, Theodotus, etc, or, transcribed, then it should be Māmās, or then two titles)

https://en.wikipedia.org/wiki/Mammes_of_Caesarea

maybe at this point we can add wd:Q532019 and life dates (259?-275?)

Reply:
I think that the English title is a little bit intransparent, "his" can refer to both persons...